app/models/user.rb

```diff
@@ -0,0 +1,25 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  validates :name, presence: true
+
+  enum role: [:user, :admin]
+  after_initialize :set_default_role, :if => :new_record?
+
+  def set_default_role
+    self.role ||= :user
+  end
+  def is_admin?
```

Comment:
with ruby it's more convenient to have `admin?` without `is_..`, also `enum` already defined such method for you, or maybe `role_admin?`

Reply:
Thanks Igor, I just checked and current_user does indeed have admin? method. I am fairly sure I was looking for that but something didn't work out. Maybe I misspelled the variable current_user somewhere. 
Anyways, it's all good now.